build.sbt

```diff
@@ -406,7 +406,6 @@ lazy val compiler = configureAsSubproject(project)
         (products in Compile in packageBin in LocalProject("interactive")).value ++
         (products in Compile in packageBin in LocalProject("scaladoc")).value ++
         (products in Compile in packageBin in LocalProject("repl")).value ++
-        (products in Compile in packageBin in LocalProject("repl-jline")).value ++
```

Comment:
Where/how do you "want to be able to use both"?

Before rewiring the package key, I was taking the fact that it was failing as proof that this setting was wrong.

But post-rewiring I have no proof, so I'm happy to drop it if you'd like.